packages/typespec-test/test/openai_non_branded/generated/typespec-ts/review/openai-non-branded.api.md

```diff
@@ -797,10 +797,16 @@ export interface OpenAIFile {
 }
 
 // @public
-export type Prompt = string | string[] | number[] | number[][];
+export type Prompt = Prompt_1 | null;
 
 // @public
-export type Stop = string | string[];
+export type Prompt_1 = string | string[] | number[] | number[][];
```

Comment:
i was thinking if we should delievy an alpha verion for keyvalut first. the correct fix without xxx_1 should be non-breaking, but this pr would introduce a breaking.

Reply:
breaking only exists when there's a real release, and as far as I know keyvault is the only one that has this issue and is going to release, and @maorleger already said they will not release with this duplicate _N suffix. And it seems like an easy fix and can be easily caught during the review stage.
However, any services that are using recursive self reference union would have the same generation crash issue in pipeline. Compare with the generation crash there, we then spend resources to investigate during this period, I prefer to have a new release out with the latest tag.
/cc @joheredi @xirzec Could you kindly keep that in mind while you review the servics onboarding PR? Thanks.